test/gtest/activation.cpp

```diff
@@ -227,7 +227,7 @@ INSTANTIATE_TEST_SUITE_P(ActivationTestSuite,
                                                 // miopenActivationLOGISTIC,
                                                 // miopenActivationTANH,
                                                 miopenActivationRELU,
-                                                miopenActivationSOFTRELU,
+                                                // miopenActivationSOFTRELU,
```

Comment:
If this is an error tolerance issue, I would recommend setting a tolerance level instead of disabling the tests.
CC: @JehandadKhan

Reply:
> If this is an error tolerance issue, I would recommend setting a tolerance level instead of disabling the tests. CC: @JehandadKhan

Yes, this is a temporary while still trying to reproduce this issue reliably to root cause.